src/routes/jira/jira-get.ts

```diff
@@ -161,6 +162,11 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 
 	const hasConnections =  !!(installations.total || gheServers?.length);
 
+	if (!hasConnections) {
+		const jiraClient = await JiraClient.getNewClient(installation, req.log);
+		await jiraClient.appPropertiesDelete();
```

Comment:
Is there a reason to delete instead of set to false? This would still work for our use case just curious :)

Reply:
The configure message shows up on either undefined or false, so doesn't really matter for this case. No specific reason for using delete here.